templates/shared/pricing/_storage-support.html

```diff
@@ -0,0 +1,54 @@
+<table>
+  <thead>
+    <td>&nbsp;</td>
+    <th class="u-align--center">Ubuntu Advantage Storage</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Up to 32 TB usable</td>
+      <td class="u-align--center">$5,000/year</td>
+    </tr>
+    <tr>
+      <td>Up to 128 TB usable</td>
+      <td class="u-align--center">$15,000/year</td>
+    </tr>
+    <tr>
+      <td>Up to 512 TB usable</td>
+      <td class="u-align--center">$45,000/year</td>
+    </tr>
+    <tr>
+      <td>Up to 1,024 TB usable (1 PB)</td>
+      <td class="u-align--center">$75,000/year</td>
+    </tr>
+    <tr>
+      <td>Unlimited storage</td>
+      <td class="u-align--center">$250,000/year</td>
+    </tr>
+    <tr>
+      <td>Phone and web ticket support</td>
+      <td class="u-align--center">24 hours a day, everyday</td>
+    </tr>
+    <tr>
+      <td>Industry-leading cloud operations tooling (Ubuntu, MAAS, Juju)</td>
+      <td class="u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
+    </tr>
+    <tr>
+      <td>Deploy, run, scale, upgrade</td>
+      <td class="u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
+    </tr>
+    <tr>
+      <td>Livepatch</td>
+      <td class="u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
+    </tr>
+    <tr>
+      <td>Knowledge Base</td>
+      <td class="u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
+    </tr>
+    <tr>
+      <td>Production grade monitoring</td>
+      <td class="u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
+    </tr>
+  </tbody>
+</table>
+
+<p><a href="https://insights.ubuntu.com/2015/05/18/ubuntu-advantage-storage-factsheet/?_ga=2.122490379.1666167933.1504515501-463981611.1499434659" class="p-link--external">Download the datasheet</a> or, <a href="/containers/contact-us?product=containers-kubernetes-foundation">contact us&nbsp;&rsaquo;</a></p>
```

Comment:
I prefer the contact us been a button and the datasheet link is a link. Like the section above.